### PR TITLE
ztp: CNF-7061: add crio wipe disable to reference configuration

### PR DESCRIPTION
**Pull request by pixelsoccupied**
/cc @imiller0 
/cc @browsell

### CODE REVIEW

File: ztp/source-crs/extra-manifest/99-crio-disable-wipe-master.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    machineconfiguration.openshift.io/role: master
+  name: 99-crio-disable-wipe-master
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - contents:
+            source: data:text/plain;charset=utf-8;base64,W2NyaW9dCmNsZWFuX3NodXRkb3duX2ZpbGUgPSAiIgo=
```
Comment (bartwensley) [inline ztp/source-crs/extra-manifest/99-crio-disable-wipe-master.yaml:15]: I think you should instead be creating this in ztp/extra-manifests-builder - that will allow you to store the file in plain text and have the extra-manifest files generated automatically.

**Review by bartwensley — CHANGES_REQUESTED** [review]

File: ztp/source-crs/extra-manifest/99-crio-disable-wipe-master.yaml (hunk 1 above)
Comment (pixelsoccupied) [reply ztp/source-crs/extra-manifest/99-crio-disable-wipe-master.yaml:15]: Ah had no idea it was thing! Will update

File: ztp/extra-manifests-builder/99-crio-disable-wipe/build.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+MCPROLE=${MCPROLE:-master}
+
+crio_disable_wipe_base64=$(base64 -w0 crio-disable-wipe.toml)
+
+echo "\
```
Comment (lack) [inline ztp/extra-manifests-builder/99-crio-disable-wipe/build.sh:7]: This works - You could cut down on a lot of the boilerplate if you use `mcmaker` like here:
https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/extra-manifests-builder/04-accelerated-container-startup/build.sh#L8-L9

**Review by lack — APPROVED** [review]

**Comment by imiller0** [comment]
/override ci/prow/e2e-gcp-ran-profile
Recent switch in CI replaces e2e-gcp-ran-profile with e2e-aws-ran-profile

**Pull request merged**